x-pack/plugins/fleet/public/applications/integrations/sections/epm/screens/detail/overview/overview.tsx

```diff
@@ -113,6 +115,31 @@ const UnverifiedCallout: React.FC = () => {
   );
 };
 
+const LS_CLOUD_POSTURE_3P_SUPPORT_WIZ_INTEGRATIONS_CALLOUT_KEY =
+  'fleet:cloudSecurityPosture:thirdPartySupport:wizIntegrationsCallout';
+
+const CloudPostureThirdPartySupportCallout = () => {
```

Comment:
What about having this component in its own file so we can add it to our team ownership? I would also pass the package name to it so that all the logic was inside the component, again to eliminate the risk smth being changed outside our our ownership. Ofc there always be the risk that the call of the component is deleted

Reply:
Sure, makes sense